conv.go

```diff
@@ -0,0 +1,196 @@
+package DateTimeMate
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	secondsPerNanoseconds = 0.000000001
+	secondsPerMicrosecond = 0.000001
+	secondsPerMillisecond = 0.001
+	secondsPerMinute      = 60
+	secondsPerHour        = 60 * secondsPerMinute
+	secondsPerDay         = 24 * secondsPerHour
+	secondsPerWeek        = 7 * secondsPerDay
+	secondsPerYear        = 365.25 * secondsPerDay
+)
+
+var unitMap = map[string]float64{
+	"nanosecond":  secondsPerNanoseconds,
+	"microsecond": secondsPerMicrosecond,
+	"millisecond": secondsPerMillisecond,
+	"second":      1,
+	"minute":      secondsPerMinute,
+	"hour":        secondsPerHour,
+	"day":         secondsPerDay,
+	"week":        secondsPerWeek,
+	"year":        secondsPerYear,
+}
+
+var unitBriefMap = map[string]string{
```

Comment:
Here is your map

https://github.com/jftuga/DateTimeMate/pull/5#discussion_r1683863526

Reply:
The `abbrevMap` map is needed for `shrinkPeriod`. The `unitBriefMap` goes in the opposite key/value direction.